graphsdk/src/androidTest/java/com/microsoft/graph/http/GraphServiceExceptionTests.java

```diff
@@ -0,0 +1,64 @@
+package com.microsoft.graph.http;
+
+import android.test.AndroidTestCase;
+
+import com.microsoft.graph.core.GraphErrorCodes;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Test cases for {@see GraphServiceException}
+ */
+public class GraphServiceExceptionTests extends AndroidTestCase {
+
+    public void testError() {
```

Comment:
I know this error message is a beast, but it would be a good min bar to check that each major component is contained in the output.  If you wanted to have a full verbose string in this test and just assertEquals(...) against the result I think that would be viable option